blocks.md

```diff
@@ -341,6 +342,32 @@ Key | Values | Required | Default
 `max_width` | Truncates titles to this length. | No | `21`
 `show_marks` | Display marks instead of the title, if there are some. Options are `"none"`, `"all"` or `"visible"`, the latter of which ignores marks that start with an underscore. | No | `"none"`
 
+## Github
+
+Creates a block which shows the unread notification count for a github account.
+
+### Examples
+
+```toml
+[[block]]
+block = "github"
+format = "{total}|{author}|{comment}|{mention}|{review_requested}"
+```
+
+### Options
+
+Key | Values | Required | Default
+----|--------|----------|--------
+`interval` | Update interval, in seconds. | No | `30`
+`format` | A format string. See below for available placeholders. | No | `"{total}"`
+
+It requires a github token, which must be passed using the `GITHUB_TOKEN` environment variable.

```

Comment:
Looks like GitHub calls them "personal access tokens" so perhaps we should align our wording with that for maximum clarity. Also might be good to mention that only the `notifications` scope is required when generating a new token for the github block.

Reply:
Alright, will do. Thanks !